### PR TITLE
Weekend at bernies now also works on any chair. Also slightly cleans up some examine code

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Cleans up some examine code slightly, and makes `Weekend_at_bernies` work on any chair, not just wheelchairs as:
1. It's already very underused
2. A lot funnier if we can do it more often (and hope that people actually notice it and read the examine text well)
image
image
Ignore the catatonic message, it is caused by me using aghost to exit the body

## Why It's Good For The Game
While keeping it to a wheelchair fits nicely with the movie, as this is an underused mechanic anyways, I don't think it would be too bad if we made any chair work for this.

## Images of changes
See two paragraphs above

## Testing
See the paragraph above

## Changelog
:cl:
tweak: Weekending at Bernie's now works with all chairs
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/examine.dm [hunk 1]
```diff
@@ -196,19 +196,16 @@
...
-					if(!foundghost)
-						msg += " and [p_their()] soul has departed"
-			msg += "...</span>\n"
+			if(get_int_organ(/obj/item/organ/internal/brain) && !key)
+				var/foundghost = FALSE
+				if(mind)
+					for(var/mob/dead/observer/G in GLOB.player_list)
+						if(G.mind == mind && G.can_reenter_corpse)
+							foundghost = TRUE
+						break
```
Comment (Contrabang) [inline code/modules/mob/living/carbon/examine.dm:205]: its gotta be breakin here
```suggestion
						if(G.mind == mind && G.can_reenter_corpse)
							foundghost = TRUE
						     break
```

Comment (DGamerL) [reply code/modules/mob/living/carbon/examine.dm]: Guh my bad, good catch!

File: code/modules/mob/living/carbon/examine.dm [hunk 2]
```diff
@@ -196,19 +196,16 @@
...
+				if(!foundghost)
+					msg += " and [p_their()] soul has departed"
+				msg += "...</span>\n"
```
Comment (Contrabang) [inline code/modules/mob/living/carbon/examine.dm:208]: This needs to show up even if there is no brain
```suggestion
			msg += "...</span>\n"
```

Comment (DGamerL) [reply code/modules/mob/living/carbon/examine.dm:208]: I am unable to count apparently.

**Comment by meow20** [comment]
I don't understand a thing about this PR lol

**Review by lewcc — APPROVED** [review]
looks good

**Review by hal9000PR — APPROVED** [review]

**Comment by spacegeese** [comment]
> I don't understand a thing about this PR lol

according to my research, putting tinted sunglasses on a dead dude makes them look like they're just sleeping, previously it only worked on wheelchairs

it's a reference to the movie Weekend at Bernies where they same thing happens to some dude, here's the original PR
18649

**Pull request merged**